plenum/server/node.py

```diff
@@ -622,7 +622,8 @@ def _create_bls_bft(self):
         return bls_bft
 
     def update_bls_key(self, new_bls_key):
-        bls_crypto_factory = create_default_bls_crypto_factory(self.basedirpath, self.name)
+        key_path = os.path.join(self.basedirpath, "keys")
+        bls_crypto_factory = create_default_bls_crypto_factory(key_path, self.name)
```

Comment:
Please also check that correct path is used in other places where BLS is initialized (for example, in `create_default_bls_bft_factory` method). Please make suer that `BlsKeyManagerFile` uses correct paths.

Reply:
fixed